website/static/js/pages/project-settings-page.js

```diff
@@ -67,14 +67,16 @@ $(document).ready(function() {
     }
     var disableCategory = !window.contextVars.node.parentExists;
     // need check because node category doesn't exist for registrations
-    if ($('#nodeCategorySettings').length) {
-        var categorySettingsVM = new ProjectSettings.NodeCategorySettings(
-            window.contextVars.node.category,
-            categories,
-            window.contextVars.node.urls.update,
-            disableCategory
-        );
-        $osf.applyBindings(categorySettingsVM, $('#nodeCategorySettings')[0]);
+    if ($('#projectSettings').length) {
+        var projectSettingsVM = new ProjectSettings.ProjectSettings( {
+            currentTitle: ctx.node.title,
+            currentDescription: ctx.node.description,
+            category: ctx.node.category,
+            categories: categories,
```

Comment:
It might help to rename something here so that the difference between `category` and `categories` is a bit more clear.